contracts/ToucanRegenBridge.sol

```diff
@@ -51,29 +51,27 @@ contract ToucanRegenBridge is Ownable, Pausable {
 
     /**
      * @dev bridge tokens to Regen Network.
-     * Burns Toucan TCO2 compatible tokens (whitelisted in ncto) and signals a
-     * bridge event.
+     * Burns Toucan TCO2 compatible tokens and signals a bridge event.
+     * @param recipient Regen address to receive the TCO2
+     * @param tco2 TCO2 address to burn
+     * @param amount TCO2 amount to burn
      */
     function bridge(
         string calldata recipient,
-        IToucanCarbonOffsets tco2,
-        uint256 amount,
-        string calldata note
```

Comment:
What was the purpose of the `note` parameter?

Reply:
Not really sure that's why I am removing it here. In case we discover though design discussions that it is needed, we can add it back.